src/main/java/com/playtika/homework4/methods/Methods.java

```diff
@@ -0,0 +1,234 @@
+package com.playtika.homework4.methods;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import com.playtika.homework4.models.CreatedNote;
+import com.playtika.homework4.models.CreatedUser;
+import com.playtika.homework4.models.Notes;
+import com.playtika.homework4.models.Token;
+import org.apache.http.HttpEntity;
+import org.apache.http.HttpResponse;
+import org.apache.http.NameValuePair;
+import org.apache.http.client.entity.UrlEncodedFormEntity;
+import org.apache.http.client.methods.HttpDelete;
+import org.apache.http.client.methods.HttpGet;
+import org.apache.http.client.methods.HttpPost;
+import org.apache.http.client.methods.HttpPut;
+import org.apache.http.entity.ContentType;
+import org.apache.http.entity.StringEntity;
+import org.apache.http.impl.client.CloseableHttpClient;
+import org.apache.http.impl.client.HttpClients;
+import org.apache.http.message.BasicNameValuePair;
+import org.apache.http.util.EntityUtils;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.InputMismatchException;
+import java.util.List;
+import java.util.Scanner;
+
+public class Methods {
+
+
+    public static CreatedUser createUser() throws IOException {
+        System.out.println("Введите e-mail");
+        Scanner sc = new Scanner(System.in);
```

Comment:
extract `Scanner sc` to the private class field and initialize it in constructor

Reply:
the same do for `HttpPost httpPost` and `ObjectMapper mapper`

Reply:
do it for all methods of this class

Reply:
Changed for 'ObjectMapper mapper' and 'Scanner sc' except 'HttpPost httpPost' because each time we pass a different value of link